CONTRIBUTING.md

```diff
@@ -18,45 +18,163 @@ any additional information for contributing code to this project.
 
 ## Building
 
-### Egg format
+### Static/portable CLI binary
 
 ```
-$ ./bin/build
+$ ./bin/build_binary
 ```
 
-### Static/portable CLI binary
+### Egg format
 
 ```
-$ ./bin/build_binary
+$ ./bin/build
 ```
 
 ## Development
 
-- Create a directory that will hold all the virtualenv packages and files:
+To setup a development environment follow the instructions in this section. Once you have done so, you will be able to see live changes made to the CLI.
+
+1. Create a directory that will hold all the virtualenv packages and files. Note that you will only need to run this command once.
+
+macOS:
+
 ```
 $ python3 -m venv venv
 ```
 
-- Enable your terminal to use those files with this command:
+Windows:
+
+```
+py -m venv venv
```

Comment:
add `$`? or is this a linux thing?

Reply:
don't know but I can add it. I think it is a pretty generic symbol to mean terminal